### PR TITLE
update Shippo type definitions

### PR DESCRIPTION
**Pull request by ejhnsn**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - https://goshippo.com/docs/reference
  - https://github.com/goshippo/shippo-node-client
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by saiichihashimoto** [comment]
Honestly, I'm not going to be able to review a 1300 line commit, especially when I made the initial shippo commit to make my build pass. Maybe someone from @goshippo can help.

**Comment by saiichihashimoto** [comment]
How do I reject reviewing this @andrewbranch ([they reviewed the original PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50393#pullrequestreview-562219877))? My original commit was purely to dump my types into DefinitelyTyped, I'm unable to commit to any work here.

**Comment by andrewbranch** [comment]
Just ignore @typescript-bot during this PR, and then later you can open another PR to remove your name from the header, or maybe @ejhnsn will do it for you as part of this one.

File: types/shippo/index.d.ts [hunk 1]
```diff
@@ -1,70 +1,135 @@
-// Type definitions for shippo 1.5
+// Type definitions for shippo 1.6
 // Project: https://github.com/goshippo/shippo-node-client
-// Definitions by: Saiichi <https://github.com/saiichihashimoto>
+// Definitions by: Saiichi <https://github.com/saiichihashimoto>, Eric Johnson <https://github.com/ejhnsn>
```
Comment (amcasey) [inline types/shippo/index.d.ts:3]: Sounds like it would be appreciated if you dropped the current owner.

**Review by amcasey — APPROVED** [review]

File: types/shippo/index.d.ts (hunk 1 above)
Comment (ejhnsn) [reply types/shippo/index.d.ts]: removed

**Pull request closed without merging**